atomic_reactor/plugins/exit_sendmail.py

```diff
@@ -104,6 +133,7 @@ def _should_send(self, rebuild, success, canceled):
         should_send_mapping = {
             self.MANUAL_SUCCESS: not rebuild and success,
             self.MANUAL_FAIL: not rebuild and not success,
+            self.MANUAL_CANCELED: not rebuild and canceled,
```

Comment:
Not fixed. Asking for "manual_canceled" but not "auto_canceled" will also give notifications for auto_canceled.